### PR TITLE
docs(root): update roadmap with recent changes and future plans

### PR DESCRIPTION
**Pull request by GCHQ-Developer-530**
## Summary of the changes

Updated roadmap with recent changes and future plans. Added popover into recently added as by the time this is on the live site, popover will be released.

## Related issue

#382

## Checklist

- [x] I have [manually accessibility tested](https://design.sis.gov.uk/accessibility/testing/manual-testing) any changes, if relevant.
- [x] I have raised this pull request against the [develop branch](https://github.com/mi6/ic-design-system/tree/develop)

### CODE REVIEW

File: src/content/structured/community/roadmap.mdx [hunk 1]
```diff
@@ -12,22 +12,67 @@ subTitle: "Our roadmap of recently released work and the work we have planned."
...
+We're getting ready to:
+
+- Investigate WCAG 2.2 changes.
+- Implement dark mode styling.
+
+As well as produce the following components:
+
+- Data visualisation.
+- Tree view.
+- Tecnhical utility components, such as no SSR and no JS.
```
Comment (GCHQ-Developer-112) [inline src/content/structured/community/roadmap.mdx:63]:
```suggestion
- Technical utility components, such as no SSR and no JS.
```

File: src/content/structured/community/roadmap.mdx [hunk 2]
```diff
@@ -12,22 +12,67 @@ subTitle: "Our roadmap of recently released work and the work we have planned."
...
-- Improve codebase of this Design System website.
-- Refresh 'patterns' section with latest user research.
+We plan to:
+
+- Investigate customisation of our components.
+- Create app templates.
+
+As well as produce the following components:
+
+- WYSIWYG editor.
```
Comment (GCHQ-Developer-112) [inline src/content/structured/community/roadmap.mdx:74]: Had to look this up
```suggestion
- What You See Is What You Get (WYSIWYG) editor.
```

File: src/content/structured/community/roadmap.mdx [hunk 3]
```diff
@@ -12,22 +12,67 @@ subTitle: "Our roadmap of recently released work and the work we have planned."
...
+- Video player.
+- Timeline.
+- Powerbar.
+- Mapping.
```
Comment (GCHQ-Developer-112) [inline src/content/structured/community/roadmap.mdx:78]:
```suggestion
- Video player,
- Timeline,
- Powerbar,
- Mapping.
```

Comment (GCHQ-Developer-530) [reply src/content/structured/community/roadmap.mdx:78]: Not sure it makes sense to end on commas as they're technically all different items.

File: src/content/structured/community/roadmap.mdx [hunk 4]
```diff
@@ -12,22 +12,67 @@ subTitle: "Our roadmap of recently released work and the work we have planned."
...
-- Research, design and build of nearly 30 web components.
-- Official communications campaign from the United Kingdom Intelligence Community (UKIC).
+We recently added:
+
+- Toasts.
+- Cards V2, with images and expandable area.
+- Asynchronous filtering for search bar.
+- Compact stepper.
+- Chips.
+- Popover menu.
```
Comment (jd239) [inline src/content/structured/community/roadmap.mdx:33]: Should these link to the relevant component pages to actually demonstrate they have been released?

**Review by jd239 — COMMENTED** [review]
Just a single comment but it's more of a suggestion. Ignore if you think the suggestion doesn't work for this case.

**Review by GCHQ-Developer-741 — APPROVED** [review]

**Comment by GCHQ-Developer-530** [comment]
> Not really an issue with this PR specifically, but I have noticed that internal links to other site pages don't work on branch deployments. For some reason it appends another "branches/branch-name" before the extension. For example, one of the links added is set to https://animated-fiesta-gvyom1k.pages.github.io/branches/382-add-updated-roadmap-to-roadmap-page/branches/382-add-updated-roadmap-to-roadmap-page/components/toasts on the deployment. They work when running locally and on the main site, so it isn't a high-priority issue, but I do wonder why its happening.

That's a weird one, might be worth raising an issue?

**Comment by GCHQ-Developer-741** [comment]
> > Not really an issue with this PR specifically, but I have noticed that internal links to other site pages don't work on branch deployments. For some reason it appends another "branches/branch-name" before the extension. For example, one of the links added is set to https://animated-fiesta-gvyom1k.pages.github.io/branches/382-add-updated-roadmap-to-roadmap-page/branches/382-add-updated-roadmap-to-roadmap-page/components/toasts on the deployment. They work when running locally and on the main site, so it isn't a high-priority issue, but I do wonder why its happening.
> 
> That's a weird one, might be worth raising an issue?

[Issue](https://github.com/mi6/ic-design-system/issues/396) raised

**Pull request merged**